plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkipping.java

```diff
@@ -118,7 +139,7 @@ private int assertSameResults(String query)
                         .setCatalogSessionProperty(getSession().getCatalog().get(), "parquet_use_column_index", "false")
                         .build(),
                 query);
-        assertEquals(withColumnIndexing, withoutColumnIndexing);
+        assertEqualsIgnoreOrder(withColumnIndexing, withoutColumnIndexing);
```

Comment:
Extract as prep commit, since it fixes pre-exsiting bug in the test.

Reply:
done